### PR TITLE
If you're jobbanned you can't join a xeno response team.

### PR DESCRIPTION
**Pull request by F-OS**
Pros: Consistency 

Cons: Literally none.

### CODE REVIEW

File: code/datums/emergency_calls/xenos.dm [hunk 1]
```diff
@@ -21,8 +21,10 @@
 /datum/emergency_call/xenos/create_member(datum/mind/M)
 	var/turf/spawn_loc = get_spawn_point()
 	var/mob/original = M.current
-
-	if(!istype(spawn_loc)) return //Didn't find a useable spawn point.
+	if(jobban_isbanned(original, "Alien"))
+		return
```
Comment (Rohesie) [inline code/datums/emergency_calls/xenos.dm]: While this does prevent xeno job-baned players from playing a XRT, it also eats up the slot if one was selected.
A way around that is to `return FALSE` here (for clarity) and then alter the logic of the loop that calls this proc, which you'll find in `/datum/emergency_call/proc/activate(announce = TRUE)`, in `emergency_call.dm`

Comment (F-OS) [reply code/datums/emergency_calls/xenos.dm]: Ah, got it, thanks again!

**Comment by Rohesie** [comment]
Do you plan on continuing this?

**Comment by F-OS** [comment]
Sorry, forgot about it, will fix soon :tm:

**Comment by F-OS** [comment]
Press :f:

**Pull request closed without merging**